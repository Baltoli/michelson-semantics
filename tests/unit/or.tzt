code { OR } ; 
input { Stack_elt int 5 ; Stack_elt int 3 } ;
output { Stack_elt int 7 }
