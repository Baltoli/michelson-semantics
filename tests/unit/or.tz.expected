<generatedTop>
  <k>
    .
  </k>
  <paramtype>
    #NotSet
  </paramtype>
  <storagetype>
    #NotSet
  </storagetype>
  <stack>
    Stack ( ListItem ( 7 ) )
  </stack>
  <balance>
    #Mutez ( 0 )
  </balance>
  <amount>
    #Mutez ( 0 )
  </amount>
  <now>
    #Timestamp ( -1 )
  </now>
  <myaddr>
    #Address ( -1 )
  </myaddr>
  <knownaddrs>
    .Map
  </knownaddrs>
  <sourceaddr>
    #Address ( -2 )
  </sourceaddr>
  <senderaddr>
    #Address ( -3 )
  </senderaddr>
  <chainid>
    #ChainId ( 0 )
  </chainid>
</generatedTop>
